src/clusterfuzz/stacktraces/constants.py

```diff
@@ -63,6 +63,10 @@
                                  r'.*assertion failed at\s.*\sin\s*.*: (.*)')
 ASSERT_REGEX_GLIBC = re.compile(
     r'.*:\s*assertion [`\'"]?(.*?)[`\'"]? failed\.?$', re.IGNORECASE)
+ASSERT_DISENGAGED_VALUE = re.compile(
+    r'.*\S.*\/.*:\d+:\s*assertion .* failed:\s*'
+    r'(optional operator.* called on a disengaged value)'
```

Comment:
This part is a little too specific. Can we make this match on anything that comes after the "assertion failed" part?

Reply:
Ah sure!
I was about to ask you about that as I noticed there are similar errors like this but with a different message.
Fixing that now.

Reply:
Yes, we should be able to catch any kind of asserts that follow this format. Not individual ones part of a particular project/code base :)

Reply:
I will change the variable/constant names to be less specific about disengaged value, too.